Source/WebKit/UIProcess/WebPageProxy.h

```diff
@@ -28,6 +28,7 @@
 #include "APIObject.h"
 #include "MessageReceiver.h"
 #include "MessageSender.h"
+#include "WebsitePoliciesData.h"
```

Comment:
Don't add more includes here.  This is widely included and we've worked hard to reduce the includes here to reduce compile time.